dali/python/nvidia/dali/_utils/hacks.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Do we really want to name it `hacks`?

Reply:
Well, what else is that?

Reply:
Workarounds?

Reply:
You work around a bug or a lack of feature. Here we try to get around _a feature_. I'd call that hacking. And c'mon, I've seen this name in several projects before.

Reply:
I still have a very mixed feelings about this....